examples/website/texture-tester/components/textures-block.js

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import CompressedTexture from './texture';
```

Comment:
Match names and file names `import Texture from 'texture';` or `import CompressedTexture from 'compressed-texture';`